### PR TITLE
Revert "Only README left"

### PR DESCRIPTION
**Pull request by borisevich-a-v**
Reverts GloryBolly/Pet_project_Djangosite#1

### CODE REVIEW

File: db.sqlite3
Comment (borisevich-a-v) [inline db.sqlite3:1]: Should we have this file here?
try to use .gitignore

File: petsite/settings.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-&j%7yw5qz1m928j0=76i=y8^!7079blo@p98*i=i)hz9zsgsu('
```
Comment (borisevich-a-v) [inline petsite/settings.py:22]: I offer to move it to env vars. You can use .env file for example

File: todolist/admin.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from django.contrib import admin
+
+from .models import *
```
Comment (borisevich-a-v) [inline todolist/admin.py:3]: asterisk syntax is considered as a bad practice. I offer to import required objects only

File: todolist/admin.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    list_display_links = ('id', 'title')
+    search_fields = ('title', 'content')
+    list_editable = ('is_published',)
+    list_filter = ('time_create', 'is_published',)
+
+
+class CategoryAdmin(admin.ModelAdmin):
+    list_display = ('id', 'name')
+    list_display_links = ('id', 'name')
+    search_fields = ('name',)
```
Comment (borisevich-a-v) [inline todolist/admin.py:17]: I would like to know what is happening here, but it's Django magic

File: todolist/models.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.urls import reverse
+
+
+class toDoList(models.Model):
```
Comment (borisevich-a-v) [inline todolist/models.py:5]: In python we usually use ThisCamelCase for classes

File: todolist/models.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    title = models.CharField(max_length=255, verbose_name="Header")
+    content = models.TextField(blank=True, verbose_name="Content")
+    photo = models.ImageField(upload_to="photos/%Y/%m/%d/", verbose_name="Photo")
+    time_create = models.DateTimeField(auto_now_add=True, verbose_name="Time of creation")
+    time_update = models.DateTimeField(auto_now=True, verbose_name="Update time")
+    is_published = models.BooleanField(default=True, verbose_name="On portal")
+    cat = models.ForeignKey('Category', on_delete=models.PROTECT, null= True)
+    def __str__(self):
```
Comment (borisevich-a-v) [inline todolist/models.py:13]: I recommend to you use `black`. It is a code formatter which will reformat your code to a nice-looking appearance.

File: todolist/tests.py [hunk 6]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (borisevich-a-v) [inline todolist/tests.py:3]: Nice tests :)

File: todolist/urls.py [hunk 7]
```diff
@@ -0,0 +1,13 @@
+from django.urls import path, re_path
+
+from .views import *
```
Comment (borisevich-a-v) [inline todolist/urls.py:3]: here

File: todolist/views.py [hunk 8]
```diff
@@ -0,0 +1,54 @@
+from django.shortcuts import render, redirect
+from django.http import HttpResponse, HttpResponseNotFound, Http404
+from .models import *
```
Comment (borisevich-a-v) [inline todolist/views.py:3]: here

File: todolist/views.py [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+    context = {'post': post,
+               'menu': menu,
+               'cats': cats,
+               'title': "Section display",
+               'cat_selected': cat_id,
+               }
+
+    return render(request, 'todolist/index.html', context=context)
+def pageNotFound(request, exception):
```
Comment (borisevich-a-v) [inline todolist/views.py:53]: In python usually snake case is used

File: todolist/models.py (hunk 5 above)
Comment (borisevich-a-v) [reply todolist/models.py:13]: You also may apply pre-commit hooks https://pre-commit.com/. It is a standard in my company.
I offer to use my config as an example https://github.com/borisevich-a-v/lang_channel/blob/master/.pre-commit-config.yaml

**Comment by borisevich-a-v** [comment]
Also every project usually contains `requirement.txt` file (or other file for dependencies).

**Comment by borisevich-a-v** [comment]
In general it looks decent, but as I said I am not good in Django stuff. Wait for fixes and then I believe I'll able to run it locally

File: todolist/urls.py [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+
+urlpatterns = [
+    path('', index, name='home'),
+    path('about/', about, name='about'),
+    path('create/', add_snippet, name='create'),
+    path('delete/', delete_snippet, name='delete'),
+    path('login/', login, name='login'),
+    path('post/<int:post_id>/', show_post, name='post'),
+    path('category/<int:cat_id>/', show_category, name='category')
```
Comment (borisevich-a-v) [inline todolist/urls.py:12]: Can you explain what these endpoints do?
The most interesting part for me is `create`, `delete`, `post` and `category.`
The reason why I'm asking is CRUD. If you have an entity and you want to create/read/update/delete it, you can just use the single endpoint for all of this operations. The only difference would be a HTTP verb (POST/GET/PUT/DELETE).

File: db.sqlite3
Comment (GloryBolly) [reply db.sqlite3:1]: done

File: todolist/admin.py (hunk 3 above)
Comment (GloryBolly) [reply todolist/admin.py:17]: it's for admin panel, just for better  looking and  filter

File: todolist/models.py (hunk 4 above)
Comment (GloryBolly) [reply todolist/models.py:5]: got u

File: todolist/admin.py (hunk 2 above)
Comment (GloryBolly) [reply todolist/admin.py:3]: Sure, it's not finished yet

**Pull request merged**